src/pages/Markets/constants.ts

```diff
@@ -1,3 +1,7 @@
+import type { MarketData } from '@shapeshiftoss/types'
+import type { SortOption } from 'components/SortDropdown/types'
+import { SortOptionsKeys } from 'components/SortDropdown/types'
+
 export enum MARKETS_CATEGORIES {
```

Comment:
I know you didn't name this, but we should make it UpperCamelCase.

```suggestion
export enum MarketCategories {
```

Reply:
Changed in [1a0f5ae](https://github.com/shapeshift/web/pull/8006/commits/1a0f5ae4b705a50d0220f5f6b637fe108ce37eee)